allennlp/tests/commands/evaluate_test.py

```diff
@@ -66,22 +66,33 @@ def test_evaluate_calculates_average_loss_with_weights(self):
     def test_evaluate_from_args(self):
         kebab_args = [
             "evaluate",
-            str(self.FIXTURES_ROOT / "bidaf" / "serialization" / "model.tar.gz"),
-            str(self.FIXTURES_ROOT / "data" / "squad.json"),
+            str(
+                self.FIXTURES_ROOT / "simple_tagger_with_span_f1" / "serialization" / "model.tar.gz"
```

Comment:
Sorry, I should have said something earlier; when picking models to replace things with in tests, we should probably just use a simple classifier.  I think all other models are going to be removed, so replacing a removed model with another model that's going to be removed just means we have more work when we go to remove the tagger.  I don't think you should worry about fixing this in this PR, it's just something to keep in mind for when you (or anyone else) does similar things in the future.

Reply:
I figured the simple tagger might stay, given its name, so I tried to put it in wherever I could.